.github/workflows/ci.yml

```diff
@@ -50,3 +56,91 @@ jobs:
         run: |
           chmod +x runTests
           ./runTests
+          
+  create_release:
+    runs-on: ubuntu-latest
+    needs: [test_course]
+    # Release gets triggered only on creating new tags
+    if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags/')
+    
+    steps:
+      
+      - name: Checkout Course Management Tools
+        uses: actions/checkout@v2
+        with:
+          repository: lightbend/course-management-tools
+          path: CMT
+          
+      - name: Checkout Course Repo
+        uses: actions/checkout@v2
+        with:
+          path: lunatech-scala-2-to-scala3-course
+          fetch-depth: 0          
+      
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      
+      - name: Cache Ivy
+        uses: actions/cache@v2
+        with:
+          path: ~/.ivy2/cache
+          key: ${{ runner.os }}-ivy--${{ hashFiles('**/build.sbt') }}
+          restore-keys: |
+            ${{ runner.os }}-ivy-
+            ${{ runner.os }}-
+      
+      - name: Cache SBT
+        uses: actions/cache@v2
+        with:
+          path: ~/.sbt            
+          key: ${{ runner.os }}-sbt-${{ hashFiles('project/**') }}
+          restore-keys: |
+            ${{ runner.os }}-sbt-            
+            ${{ runner.os }}-
+      
+      - name: Studentify Repo
+        run: |
+          mkdir -p studentified
+          (cd CMT/ && exec sbt "studentify -dot ../lunatech-scala-2-to-scala3-course ../studentified")
+          (cd studentified && exec zip -r lunatech-scala-2-to-scala3-course.zip lunatech-scala-2-to-scala3-course)  
+          
+      - name: Linearize Repo
+        run: |
+          mkdir -p linearized
+          git config --global user.email "no-reply@lunatech.com"
+          git config --global user.name "Lunatech Labs"
+          (cd CMT/ && exec sbt "linearize -dot ../lunatech-scala-2-to-scala3-course ../linearized")
+          (cd linearized && exec zip -r lunatech-scala-2-to-scala3-course.lin.zip lunatech-scala-2-to-scala3-course)
```

Comment:
Could this be changed so that instead of having `lunatech-scala-2-to-scala3-course` as the root folder in the zipped linearized repo, it is `lunatech-scala-2-to-scala3-course-linearized`?
The reason is that, when people download both the studentified and the linearized repo zip files, both unzip to the same base folder. Changing the name of the linearized version solves this.